### DISCUSSION TITLE
Support for Windows Containers

### DISCUSSION THREAD

**Discussion by ksl28** [containers/podman#19492] [Ideas]
### Feature request description

Based on this comment https://github.com/containers/podman/issues/18419#issuecomment-1532918353 and also tested it, it seems that Podman does not support Windows Containers.
While many containers are based on an UNIX OS, we have several containers that are running Windows (Windows IIS, Powershell Universal on Windows, etc) - and more and more are coming.

This is a major blocker for us, and is preventing us from moving to Podman.

### Suggest potential solution

Add support for Windows Containers

### Have you considered any alternatives?

Use Docker - but thats not really an alternative :)

**Comment by Luap99** [comment]
Duplicate of #11809

If someone has serious motivation to port Podman then I think we are open to contributions but that would mean signing up for maintenance in the long term as well. This would be a lot of work and the core team is linux only has no interest in windows containers.

**Comment by ksl28** [comment]
Ohh! I was under the impression, that Podman / Docker executed the same type of command to run the container, with no knowledge of the OS in the container.

**Comment by vrothberg** [comment]
Converted to a discussion.  As @Luap99 mentioned, there are no plans to support Windows container natively.
